### ISSUE TITLE
Object.values is not defined

### ISSUE THREAD

**Issue by bradzacher** [join-monster/join-monster#144]
I updated from v1.2.1-beta.9 to v1.2.6-beta.1 and JM seems to be broken on node < v7.

Looks like something went wrong with the babel transpilation?

```
TypeError: Object.values is not a function
    at /var/task/build/node_modules/join-monster/dist/batch-planner/index.js:35:29
    at next (native)
    at step (/var/task/build/node_modules/join-monster/dist/batch-planner/index.js:19:191)
    at /var/task/build/node_modules/join-monster/dist/batch-planner/index.js:19:437
    at /var/task/build/node_modules/join-monster/dist/batch-planner/index.js:19:99
    at nextBatch (/var/task/build/node_modules/join-monster/dist/batch-planner/index.js:138:17)
    at Object.<anonymous> (/var/task/build/node_modules/join-monster/dist/index.js:17:38)
    at next (native)
    at step (/var/task/build/node_modules/join-monster/dist/index.js:103:191)
    at /var/task/build/node_modules/join-monster/dist/index.js:103:361
```

**Comment by gemabarni** [comment]
I was just about to post this same issue. It would be nice to solve since this error makes it hard to work with AWS Lambda.

**Comment by acarl005** [comment]
That's odd. The `.babelrc` is using `babel-preset-node6`. Maybe Travis-CI borked the publishing process. Will look into this tonight. I should also probably set up the test suite to run on multiple node versions.

**Comment by gemabarni** [comment]
It seems to me that `babel-preset-node6` does not contain transpilation for `Object.values()`, maybe it can be solved with `babel-plugin-object-values-to-object-keys` or just by simply using `Object.keys(obj).map(key => obj[key])` instead of `Object.values()`.

**Comment by acarl005** [comment]
Should now be fixed in **v1.2.7-beta.1**!
